cmdstanpy/utils.py

```diff
@@ -1020,6 +1020,25 @@ def install_cmdstan(
     return True
 
 
+def flatten_chains(draws_array: np.ndarray) -> np.ndarray:
```

Comment:
I am happy with this in Utils. It may be nice to write a function which goes the other way both for round-trip testing and for some users who really dig into the api.